torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12340,6 +12340,21 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                DecorateInfo(unittest.expectedFailure, 'TestMeta', 'test_dispatch_meta_outplace', device_type='cuda'),
                DecorateInfo(unittest.expectedFailure, 'TestMeta', 'test_dispatch_symbolic_meta', device_type='cuda'),
            )),
+    OpInfo('view_copy',
+           dtypes=all_types_and(torch.bool, torch.bfloat16, torch.float16, torch.chalf),
```

Comment:
I think you should remove `chalf` from here. It is unlikely that op supports `chalf` but doesn't support other complex dtypes.

Reply:
removed